src/storage-preview/azext_storage_preview/_transformers.py

```diff
@@ -122,5 +124,67 @@ def transform_storage_list_output(result):
     return list(result)
 
 
+# pylint: disable=unused-argument
 def transform_file_upload(result):
     return None
+
+
+def transform_metadata(result):
+    return result.metadata
+
+
+def transform_queue_stats_output(result):
+    if type(result).__name__ == 'dict':
+        new_result = {}
+        for key in result.keys():
+            new_key = convert_to_camel_case(key)
```

Comment:
utilize existing function in azure.cli.core.
https://github.com/Azure/azure-cli/blob/e4906ced2b2b647326ba095b67b37e91b2f92f19/src/azure-cli-core/azure/cli/core/commands/arm.py#L970-L974

Reply:
done